frontend/src/ImageUpload/ImageUpload.tsx

```diff
@@ -0,0 +1,163 @@
+import React from "react";
+import { Button, IconButton } from "@material-ui/core";
+import {
+  makeStyles,
+  createStyles,
+  Theme,
+  useTheme,
+} from "@material-ui/core/styles";
+import { FirebaseStorage } from "../firebase/index";
+import "firebase/storage";
+import firebase from "firebase/app";
+import HighlightOffOutlinedIcon from "@material-ui/icons/HighlightOffOutlined";
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      width: '100%'
+    },
+    imageContainer: {
+      display: 'flex',
+      flexDirection: "row",
+      width: '100%',
+      flexWrap: "wrap"
+    },
+    imageCard: {
+      display: 'flex',
+      width: '45%',
+      height: "30%",
+      flexDirection: "row-reverse",
+      position: 'relative',
+      margin: "0 auto",
+      padding: "5px"
+    },
+    uploadButton: {
+      width: '100%',
+      margin: '15px auto'
+    },
+    muralImage: {
+      borderRadius: '10px',
+      width: '100%',
+      height: '120px'
+    },
+    deleteButton: {
+      position: 'absolute',
+      padding: theme.spacing(1)
+    }
+  })
+);
+
+interface Image {
+  url: string;
+  path: string;
+}
+
+interface IImageUpload {
+  uploadHandler: (url: string, path: string) => void;
+  removeHandler: (path: string) => void;
+  imgsUrlAndPath: Image[];
+}
+
+function ImageUpload({
+  uploadHandler,
+  removeHandler,
+  imgsUrlAndPath,
+}: IImageUpload) {
+  const styles = useStyles();
+  const theme = useTheme();
```

Comment:
there's one error about this from Vercel preventing deployment:

![image](https://user-images.githubusercontent.com/36117635/108854734-2fb7bb80-75b6-11eb-93bd-0248e85ca671.png)